fe1-web/parts/lao/socialMedia/SocialProfile.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { Text, View } from 'react-native';
+import STRINGS from 'res/strings';
+
+const SocialProfile = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
```

Comment:
Same comment about style sheet. By the way since it is the same everywhere, maybe you can declare it somewhere and export it to other files, to avoid duplication.

Reply:
Thanks for the advice! But I will delete them for simplicity, we don't need them here.